puppet/modules/web/manifests/vhost/web.pp

```diff
@@ -66,8 +66,8 @@
   ]
 
   web::vhost { 'web':
-    servername    => 'theforeman.org',
-    serveraliases => ['www.theforeman.org'],
+    servername    => "web-backend.${facts['networking']['fqdn']}",
+    serveraliases => ['theforeman.org', 'www.theforeman.org'],
```

Comment:
this is the same logic I did in #2124, with the long-term hope to move this thing to the CDN and not have those aliases pointing at the actual web machine anymore.